spesbot_description/urdf/spesbot.xacro

```diff
@@ -42,6 +42,16 @@
             <command_interface name="position" />
             <state_interface name="position" />
         </joint>
+    </ros2_control>
+    
+    <ros2_control name="isaac_system" type="system">
+
+        <hardware>
+            <plugin>topic_based_ros2_control/TopicBasedSystem</plugin>
+            <param name="joint_commands_topic">/joint_command</param>
+            <param name="joint_states_topic">/joint_states</param>
```

Comment:
```suggestion
            <param name="joint_commands_topic">/isaac/joint_command</param>
            <param name="joint_states_topic">/isaac/joint_states</param>
```
Isolate these topics into `/isaac` namespace so they don't get mixed with the joint_state_publisher (you will need to change the topic names in Isaac as well)